lib/mbta.js

```diff
@@ -1,11 +1,39 @@
 'use strict'
 
-// Code here.
+let red = ['South Station','Park Street','Kendall','Central','Harvard','Porter','Davis','Alewife']
+let green = ['Government Center','Park Street','Boylston','Arlington','Copley','Hynes,Kenmore']
+let orange = ['North Station','Haymarket','Park Street','State','Downtown Crossing','Chinatown','Backbay','Forest Hills']
+
+let mbta = {}
+
+mbta.red = red
+mbta.green = green
+mbta.orange = orange
```

Comment:
You want your `mbta` object to include all information about the lines, stops included. Something like this: `const subwayLines = {
  Red: [
    'South Station',
    'Park Street',
    'Kendall',
    'Central',
    'Harvard',
    'Porter',
    'Davis',
    'Alewife'
  ],
  Green: [
    'Government Center',
    'Park Street',
    'Boylston',
    'Arlington',
    'Copley',
    'Hynes',
    'Kenmore'
  ],
  Orange: [
    'North Station',
    'Haymarket',
    'Park Street',
    'State',
    'Downtown Crossing',
    'Chinatown',
    'Back Bay',
    'Forest Hills'
  ]
}`